wrangler-core/src/e2e-test/java/io/cdap/plugin/wrangler/stepsdesign/Wrangler.java

```diff
@@ -19,10 +19,7 @@
 import io.cdap.e2e.pages.actions.CdfConnectionActions;
 import io.cdap.e2e.pages.actions.CdfStudioActions;
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
-import io.cdap.e2e.utils.CdfHelper;
-import io.cdap.e2e.utils.ElementHelper;
-import io.cdap.e2e.utils.PluginPropertyUtils;
-import io.cdap.e2e.utils.WaitHelper;
+import io.cdap.e2e.utils.*;
```

Comment:
Please don't use *. Use import individually, it won't give Checkstyle?

Reply:
resolved